docs/guides-custom-deploy.md

```diff
@@ -0,0 +1,206 @@
+---
+id: guides-custom-deploy
+title: Custom Aragon Organization Deployment using aragonCLI
+sidebar_label: Custom deploy with aragonCLI
+---
+
+This guide will show you the process to deploy a blank organization and configure it as a custom organization for a cooperative.
+
+We assume you have a general understanding of Aragon stack.
+
+In order to get started you will need the latest version of the CLI installed so open a terminal and enter the following:
+
+
+```sh
+npm install -g @aragon/cli
+
+```
+
+## General considerations
+
+In order to deploy to rinkeby or mainnet using the CLI you will either need to use a hardware wallet with [Frame](https://frame.sh) or configure aragonCLI to [sign transactions using a private key](/docs/cli-faq.html#set-a-private-key).
+
+If you decide to use frame, just add `--use-frame` to any of these commands.
+
+For any deploy always remember to have in a separate terminal the [IPFS](https://docs.ipfs.io/introduction/overview/) daemon running, use `aragon ipfs` to start it.
+
+If you are testing locally:
+
+- In a separate terminal run: `aragon devchain`
+
+If you are deploying to rinkeby:
+
+- Add `--environment aragon:rinkeby` to each of these commands
+
+If you are deploying to mainnet:
+
+- Add `--environment aragon:mainnet` to each of these commands
+
+## Creating a blank organization
+
+```sh
+dao new
+```
+
+This will create a new organization based on default [`bare-kit`](https://github.com/aragon/dao-kits/blob/master/kits/bare/contracts/BareKit.sol) it won’t have a token manager, vault, finance, or voting instance installed. Make a note of the deployed DAO address as we will need it for subsequent commands.
```

Comment:
```suggestion
This will create a new organization based on the default [`bare-kit`](https://github.com/aragon/dao-kits/blob/master/kits/bare/contracts/BareKit.sol). It won’t have a token manager, vault, finance, or voting instance installed. Make a note of the deployed DAO address as we will need it for subsequent commands.
```